app/src/main/res/values/strings.xml

```diff
@@ -339,11 +339,11 @@
     <string name="battery">Battery</string>
     <string name="liveactivity_live_activity">Live activity</string>
     <string name="weeksteps_today_steps_description">Steps today, target: %1$s</string>
-    <string name="pref_title_dont_ack_transfer">Do not ack activity data transfer</string>
+    <string name="pref_title_dont_ack_transfer">Do not ACK activity data transfer</string>
     <string name="pref_summary_dont_ack_transfers">If the activity data are not acked to the band, they will not be cleared. Useful if GB is used together with other apps.</string>
     <string name="pref_summary_keep_data_on_device">Will keep activity data on the Mi Band even after synchronization. Useful if GB is used together with other apps.</string>
-    <string name="pref_title_low_latency_fw_update">Use low-latency mode for Firmware updates</string>
-    <string name="pref_summary_low_latency_fw_update">This might help on devices where firmware updates fail</string>
+    <string name="pref_title_low_latency_fw_update">Use low-latency mode for firmware upgrades</string>
+    <string name="pref_summary_low_latency_fw_update">This might help on devices where firmware upgrades fail</string>
```

Comment:
Maybe we should use "installation" which is neutral regarding downgrades

Reply:
Changes?